package.json

```diff
@@ -91,11 +93,11 @@
     "zx": "^1.14.1"
   },
   "peerDependencies": {
-    "@storybook/addons": "6.4.0-beta.11",
-    "@storybook/api": "6.4.0-beta.11",
-    "@storybook/components": "6.4.0-beta.11",
-    "@storybook/core-events": "6.4.0-beta.11",
-    "@storybook/theming": "6.4.0-beta.11",
+    "@storybook/addons": "^6 || ^6.4.0-beta",
+    "@storybook/api": "^6 || ^6.4.0-beta",
+    "@storybook/components": "^6 || ^6.4.0-beta",
+    "@storybook/core-events": "^6 || ^6.4.0-beta",
+    "@storybook/theming": "^6 || ^6.4.0-beta",
```

Comment:
`^6.4.0-beta` doesn't match anything unfortunately

Reply:
ok I just updated it to `^6 || 6.4.0-rc.1` what do you think?